water.css

```diff
@@ -3,11 +3,22 @@ body {
   color: #dbdbdb;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   text-rendering: optimizeLegibility;
-  margin: 20px;
+  margin: 20px auto;
   max-width: 800px;
   line-height: 1.4;
 }
 
+button, input[type="submit"] {
+  cursor: pointer;
```

Comment:
This selector is declared twice. I'd move `cursor: pointer;` down to the selector on line 71.